### ISSUE TITLE
Can't fulfil dependency libzip2 on Ubuntu 15.10

### ISSUE THREAD

**Issue by JonathonAshworth** [mgba-emu/mgba#265]
```
Unpacking libmgba (0.5-3616-210ef6d) ...
dpkg: dependency problems prevent configuration of libmgba:
 libmgba depends on libzip2; however:
  Package libzip2 is not installed.
```

Installing the latest nightly build, I get this error.

libzip2 doesn't appear to be in the software repository for version 15.10 of Ubuntu. Any idea how to fix?

**Comment by endrift** [comment]
It needs to be libzip4 on 15.10 and newer, but apparently I haven't detected that in the makefile.

**Comment by sergiobenrocha2** [comment]
@JonathonAshworth why don't you use the PPA?

**Comment by JonathonAshworth** [comment]
I didn't know one existed, it wasn't mentioned on the downloads page of the website, I just assumed the .deb file was the standard way of installing.

**Comment by JonathonAshworth** [comment]
Installing from the ppa works, thanks :), should I leave this open so the download can be fixed for the next release?

**Comment by endrift** [comment]
The PPA doesn't have the version strings working properly, but otherwise it's pretty much the same. It's out of date sometimes, though. If I could figure out how to solve those two things, the PPA would be the ideal way to install on Ubuntu. As it stands though, the version string issue is pretty bad for a nightly, so I'm not recommending it at the moment.

Also yes, please leave this open. I'll have a fix prepared for the issue tonight.

**Comment by endrift** [comment]
Fixed in e08087a682c011bd8893803939b67419fdd61b18.
